composer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "webdevstudios/wd_s",
+    "description": "A theme boilerplate for WebDevStudios.",
+    "type": "project",
```

Comment:
@gregrickaby One of the accepted project types is `wordpress-theme`. If a project is using the Composer Installers package, they can specify that packages of type `wordpress-theme` get installed in a particular directory.